.github/labeler.yml

```diff
@@ -1,40 +1,36 @@
 Documentation:
 - changed-files:
-  - any-glob-to-any-file: docs/**
-
-NuGet:
-- changed-files:
-  - any-glob-to-any-file: '**/*.nuspec'
+  - any-glob-to-any-file: docs/** # Any modifications to Documentation
+  - any-glob-to-any-file: EXILED/Localization/** # Any modifications to Localization
 
-Scripts: # Add the 'Scripts' label
+GitHub:
 - changed-files:
+  - any-glob-to-any-file: '**/*.nuspec' #NuGet
   - any-glob-to-any-file: '**/*.ps1' # Windows
   - any-glob-to-any-file: '**/*.sh' # Linux
+  - any-glob-to-any-file: .github/** # Any modifications to github related files
 
-regarding-events: # Add the 'Events' label
+Events: # Add the 'Events' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.Events/** # Any Modifications to events
+  - all-globs-to-all-files: '!EXILED/Exiled.Events/Patches/**/*' # Exept modifications to transpiler files
 
-regarding-api: # Add the 'API' label
+API: # Add the 'API' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.API/** # Any modifications to the API
 
-regarding-transpiler: # Add the 'transpiler' label
+Transpiler: # Add the 'transpiler' label
```

Comment:
Why is this being modified?

The names are the labels which were set for consistency with older labels.

Reply:
We are going to modify label name after this merge